src/java/org/apache/cassandra/index/sai/disk/v3/CassandraDiskAnn.java

```diff
@@ -141,28 +141,36 @@ public CloseableIterator<ScoredRowId> search(float[] queryVector,
     {
         CassandraOnHeapGraph.validateIndexable(queryVector, similarityFunction);
 
-        // Retrieve the view reference once.
-        var threadLocalView = view.get();
+        var view = graph.getView();
```

Comment:
try-with-resources here?

Reply:
Because of resumable search, we can't close eagerly.

Reply:
Though I do need to make sure I close this in the event of an exception. Cleaning up now.